cfgov/unprocessed/js/organisms/MortgagePerformanceTrends/apps/map.js

```diff
@@ -146,10 +146,10 @@ MortgagePerformanceMap.prototype.renderChart = function( prevState, state ) {
   const currId = state.geo.id;
   let zoomLevel;
   if ( !utils.isDateValid( state.date, this.endDate ) ) {
-    utils.showEl( this.$notification );
+    this.$notification.classList.add( 'm-notification__visible' );
     return;
   }
-  utils.hideEl( this.$notification );
+  this.$notification.classList.remove( 'm-notification__visible' );
```

Comment:
Doesn't have to happen here, but we should strip out that leading `$`, which is a jquery object convention, which this isn't.

Reply:
Yeah, I was surprised to see that too. Before we strip it out in another PR, we should check with @contolini to see if there's a reason behind it.

Reply:
That's just me living in 1995.